libdeno/binding.cc

```diff
@@ -83,6 +83,66 @@ std::string EncodeExceptionAsJSON(v8::Local<v8::Context> context,
   // auto exception_str = message->Get();
   CHECK(json_obj->Set(context, v8_str("message"), exception_str).FromJust());
 
+  auto maybe_source_line = message->GetSourceLine(context);
```

Comment:
This whole section is unfortunately quite un-DRY. Maybe we can refactor it later somehow (an X macro maybe?)

Reply:
I tried it at first with a macro but it seemed more complexity than it was worth.